### PR TITLE
Added skeleton for side panels (MC side panel & CAP side panel)

### LINKED ISSUE #116: Add side panel to UI displaying total MCs and CAP information

### PR DESCRIPTION
**Pull request by dargohzy**

### CODE REVIEW

File: src/main/java/igrad/logic/commands/SelectAvatarCommand.java [hunk 1]
```diff
@@ -11,7 +11,8 @@
  */
 public class SelectAvatarCommand extends Command {
 
-    public static final String MESSAGE_SUCCESS = "You have chosen an animal guide!";
+    public final String MESSAGE_SUCCESS;
```
Comment (yjskrs) [inline src/main/java/igrad/logic/commands/SelectAvatarCommand.java:14]: non-final message shouldn't be snake case all caps! can consider making it camel case non-final or removing this field completely

Comment (nathanaelseen) [reply src/main/java/igrad/logic/commands/SelectAvatarCommand.java:14]: oh I've changed it back, no worries!

Comment (nathanaelseen) [reply src/main/java/igrad/logic/commands/SelectAvatarCommand.java:14]: actually @dargohzy, you could also declare placeholders in final strings, I'm changing this line to;
`public final String MESSAGE_SUCCESS = "Hi, I'm %s, let's get started!\n" + MESSAGE_ADD_COURSE;`

File: src/main/java/igrad/ui/AvatarSelectionPanel.java [hunk 2]
```diff
@@ -61,7 +64,9 @@ private void initAvatarImgList() {
 
         for (int i = 0; i < 2; i++) {
             for (int j = 0; j < 3; j++) {
-                AvatarImage avatarImg = new AvatarImage("/avatars/" + avatarList.get(index).getName() + ".png", i, j);
+                String avatarName = avatarList.get(index).getName();
```
Comment (nathanaelseen) [inline src/main/java/igrad/ui/AvatarSelectionPanel.java:67]: good modularisation 👍

File: src/main/java/igrad/ui/AvatarSelectionPanel.java [hunk 3]
```diff
@@ -74,17 +79,23 @@ private void initAvatarImgList() {
...
-
+            avatarDisplay.setFitHeight(0.12 * primaryScreenBounds.getHeight());
+            avatarDisplay.setFitWidth(0.12 * primaryScreenBounds.getHeight());
             avatarDisplay.setImage(avatarImg);
-            GridPane.setRowIndex(avatarDisplay, avatarImg.getRowIndex());
-            GridPane.setColumnIndex(avatarDisplay, avatarImg.getColIndex());
-            avatarGridPane.getChildren().add(avatarDisplay);
+
+            avatarName.setStyle("-fx-font-family: 'Calibri Light'; -fx-font-size: 30; -fx-text-fill: white");
+            avatarName.setPadding(new Insets(200, 0, 0, 0));
```
Comment (nathanaelseen) [inline src/main/java/igrad/ui/AvatarSelectionPanel.java:92]: @dargohzy I'm not sure probably the padding set by this line was too much, and there's not enough space to display all the uwus (as it appeared on my screen). Reducing this helped, to fit all the uwus, but the animal names were all too big, please fix.

File: src/main/java/igrad/ui/CommandBox.java [hunk 4]
```diff
@@ -22,9 +24,14 @@
 
     private final CommandExecutor commandExecutor;
 
+    private static String animalGuideIcon;
+
     @FXML
     private TextField commandTextField;
 
+    @FXML
+    private ImageView animalGuide;
```
Comment (nathanaelseen) [inline src/main/java/igrad/ui/CommandBox.java:33]: this line's prob redundant, there's no such GUI component in the `CommandBox.fxml` file, I'm removing it. Actually the ImageView to display the Uwu in the command box is in `ResultDisplay.fxml` file instead.

File: src/main/java/igrad/ui/MainWindow.java [hunk 5]
```diff
@@ -127,13 +135,15 @@ void displayAvatarSelectionPanel(Model model) {
         resultDisplayPlaceholder.getChildren().add(resultDisplay.getRoot());
         resultDisplay.setFeedbackToUser("Choose an animal guide by entering the NAME of the animal");
 
-        displayStatusPanels(model);
+        displayStatusBar(model);
```
Comment (nathanaelseen) [inline src/main/java/igrad/ui/MainWindow.java:138]: I'm removing the status bar, not necessary when selecting the Uwu in the Uwu selection screen

File: src/main/java/igrad/ui/MainWindow.java [hunk 6]
```diff
@@ -148,21 +158,36 @@ void displayModulePanel(Model model) {
...
+    }
 
+    /**
+     * Fills up and displays the placeholder of the command box.
+     */
+    void displayCommandBox(Model model) {
         CommandBox commandBox = new CommandBox(c -> executeCommand(c, model));
         commandBoxPlaceholder.getChildren().add(commandBox.getRoot());
-
-        mcCount.setText("MCs:\n40/160");
```
Comment (nathanaelseen) [inline src/main/java/igrad/ui/MainWindow.java:165]: ok

File: src/main/java/igrad/ui/StatusBar.java [hunk 7]
```diff
@@ -19,20 +19,20 @@
...
         course.setText("Course: Computer Science");
-        capMcDisplay.setText("CAP: 4.24     Target: 4.00     Required: 3.90/Sem");
 
         StackPane.setAlignment(course, Pos.CENTER_LEFT);
-        StackPane.setAlignment(capMcDisplay, Pos.CENTER_RIGHT);
-        pane.getChildren().addAll(course, capMcDisplay);
+        pane.getChildren().add(course);
+    }
+
+    public void setCourseName(String courseName) {
```
Comment (nathanaelseen) [inline src/main/java/igrad/ui/StatusBar.java:34]: ok thanks!

**Review by nathanaelseen — APPROVED** [review]

**Pull request merged**